src/main/java/core/basesyntax/model/FruitRecord.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.model;
+
+public class FruitRecord {
+    private final Operation operation;
+    private final String fruitName;
```

Comment:
lets create some Fruit class in case wi will have more Fruit parameters in the future

Reply:
fixed